2-WebApp-graph-user/2-5-HybridFlow/wwwroot/js/HybridSPA.js

```diff
@@ -0,0 +1,101 @@
+﻿function getTokenFromCache() {
+    const [account] = msalInstance.getAllAccounts();
+    const scopes = ["user.read"];
+    return msalInstance.acquireTokenSilent({
+        account,
+        scopes
+    });
+};
```

Comment:
Because we're getting access tokens from an auth code provided by a server which is then stored in the client cache this app assumes that all tokens are stored in the client side cache and will not prompt users for credentials.